lib/features/reps_core/reps_core.module

```diff
@@ -268,6 +268,22 @@ function reps_core_block_view($delta = '') {
   return $block;
 }
 
+/**
+ * Implements hook_views_query_alter().
```

Comment:
The hook should be place in  MODULENAME.views.inc

Reply:
ok